lib/kanta/translations/messages/finders/get_message.ex

```diff
@@ -35,6 +39,7 @@ defmodule Kanta.Translations.Messages.Finders.GetMessage do
     |> filter_query(params[:filter])
     |> search_query(params[:search])
     |> preload_resources(params[:preloads] || [])
+    |> limit(1)
```

Comment:
@vi0dine 
This is also an issue up the stack where it doesn't provide a valid id filter at some point. Just the `limit 1` seems to be good enough to prevent `Repo.one` from crashing from > 1 records given.